plugins/meta/portmap/main.go

```diff
@@ -164,7 +164,10 @@ func parseConfig(stdin []byte, ifName string) (*PortMapConf, error) {
 			}
 
 			// Skip known non-sandbox interfaces
-			intIdx := ip.Interface
+			if ip.Interface == nil {
```

Comment:
@squeed yeah, it would be.  maybe we should upconvert and set index to 0, since there will only ever be a single interface for 0.2.0?